docstr_coverage/coverage.py

```diff
@@ -381,13 +419,22 @@ def _execute():
     if len(filenames) < 1:
         sys.exit("No Python files found")
 
+    ignore_names = ()
+
+    if options.ignore_names_file == '.docstr_coverage':
+        options.ignore_names_file = Path(path, options.ignore_names_file)
+
+    if os.path.isfile(options.ignore_names_file):
+        ignore_names = tuple([line.split() for line in open(options.ignore_names_file).readlines() if ' ' in line])
```

Comment:
Yes, there is a reason for casting as a tuple, although it is not so important.
Default value for function parameters shouldn't be mutable. It is why the default value of ignore_names is an empty tuple.
Because of that I was (and still am) thinking that it is good to have ignore_names parameter as only one type (a tuple). The other way is to set ignore_names to None as default and checking it every time when it is used.